<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tips</class>
 <widget class="QWidget" name="Tips">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавить чаевые</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	border:none;
	background-color: #666699;
}</string>
  </property>
  <widget class="QWidget" name="TipsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>15</y>
     <width>350</width>
     <height>570</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	border: none;
	background-color: #333333;
	border-radius: 4px;
}</string>
   </property>
   <widget class="QPushButton" name="tip5per">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	border: none;
	background-color: #000033;
	border-radius: 7px;
}


QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>5%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tip10per">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	border: none;
	background-color: #000033;
	border-radius: 7px;
}


QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>10%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tip15per">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	border: none;
	background-color: #000033;
	border-radius: 7px;
}


QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>15%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tip20per">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	border: none;
	background-color: #000033;
	border-radius: 7px;
}


QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>20%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tip25per">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	border: none;
	background-color: #000033;
	border-radius: 7px;
}


QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>25%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tipthissum">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	border: none;
	background-color: #000033;
	border-radius: 7px;
}


QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>Конкретная
сумма</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tipthisper">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>63</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	border: none;
	background-color: #000033;
	border-radius: 7px;
}


QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>Процент</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid #000033;
	color:	black;
	border-radius: 7px;
	background-color: #edf5ef;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Чаевые (%/₽)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
